package.json

```diff
@@ -95,7 +95,7 @@
     "eth-ens-namehash": "^2.0.8",
     "eth-hd-keyring": "^1.2.2",
     "eth-json-rpc-filters": "^3.0.4",
-    "eth-json-rpc-infura": "^3.2.0",
+    "eth-json-rpc-infura": "github:brave-experiments/eth-json-rpc-infura#master",
```

Comment:
For now the project id can be hardcoded from in here for testing this PR

Reply:
I don't think you need to specify the branch name if it's master and master is the default branch